dist/resources/js/StateController.js

```diff
@@ -325,7 +367,7 @@ SequraFE.appPages = {
         const getControllerConfiguration = (controllerName, page) => {
             let config = utilities.cloneObject(configuration.pageConfiguration[controllerName] || {});
             Object.keys(config).forEach((key) => {
-                config[key] = config[key].replace(encodeURIComponent('{storeId}'), this.getStoreId);
+                config[key] = config[key].replace('{storeId}', this.getStoreId);
```

Comment:
```suggestion
                config[key] = config[key].replace('{storeId}', encodeURIComponent(this.getStoreId));
```
Just in case in any platform the getStoreId is a string and could include especials chars like '+' or '/'

Reply:
I'm applying the fix to the file in `src/` because the one in `dist/` shouldn't be modify as it's the production ready version which is generated automatically